src/library_path.js

```diff
@@ -67,6 +67,8 @@ mergeInto(LibraryManager.library, {
     basename: function(path) {
       // EMSCRIPTEN return '/'' for '/', not an empty string
       if (path === '/') return '/';
+      path = PATH.normalize(path);
+      path = path.replace(/\/$/, "");
```

Comment:
Hmm so normalize already remove trailing slash?  I don't if it should?

Reply:
Normalize doesn't remove the trailing slash, it looks like it retains both leading and trailing slashes if any, and reduces any redundant double/triple/several adjacent slashes like `/emscripten///` will just become `/emscripten/`.